docs/contributors_guide/maintenance/iso_release_procedures/Pre_Release_Checklist.rst

```diff
@@ -1,33 +1,18 @@
 Pre-Release Checklist
 =====================
 
-The bulk of the work to release both CentOS 6 and CentOS 7 versions of
+The bulk of the work to release both :term:`EL` 6 and :term:`EL` 7 versions of
 a SIMP ISO is to verify that each ISO is ready for release. Below is
-the list of verifications that must be executed *for each ISO*, before
+the list of verifications that must be executed **for each ISO**, before
 proceeding with the release of that ISO. If any of these steps fail,
 the problem identified must be fixed before you can proceed with the tag
 and release steps.
 
-* `Verify RPMs are available in packagecloud`_
-* `Verify a valid Puppetfile exists`_
-* `Verify the Changelog.rst`_
-* `Verify the dependencies.yaml`_
-* `Verify the simp-core RPMs can be created`_
-* `Verify simp-core tests pass`_
-* `Verify ISOs can be created`_
-* `Verify SIMP ISO boot options work`_
-* `Verify component interoperability`_
-* `Verify otherwise untested capabilities`_
-* `Verify SIMP server is compliant with security standards`_
-* `Verify SIMP server RPM install`_
-* `Verify SIMP server RPM upgrade`_
-* `Verify SIMP server R10K install`_
-
-Verify RPMs are available in packagecloud
+Verify RPMs are available in PackageCloud
```

Comment:
Why was this gutted?

Reply:
Too much repetition to keep up. Will try to replace it with a TOC